nodes/basic-pow/Cargo.toml

```diff
@@ -0,0 +1,93 @@
+[build-dependencies]
+vergen = '3.0.4'
+
+[build-dependencies.build-script-utils]
+git = 'https://github.com/paritytech/substrate.git'
+package = 'substrate-build-script-utils'
+tag = 'v2.0.0-alpha.3'
+
+[package]
+authors = ['Anonymous']
+build = 'build.rs'
+edition = '2018'
+homepage = 'https://substrate.dev'
+license = 'Unlicense'
+name = 'basic-pow'
+repository = 'https://github.com/paritytech/substrate/'
+version = "2.0.0"
+
+[[bin]]
+name = 'basic-pow'
+path = 'src/main.rs'
+
+[dependencies]
+futures = '0.3.1'
+log = '0.4.8'
+structopt = '0.3.8'
+parity-scale-codec = '1.0.0'
+sha3 = "0.8"
+rand = { version = "0.7", features = ["small_rng"] }
+sc-consensus-pow = { git = "https://github.com/paritytech/substrate", tag = 'v2.0.0-alpha.3' }
+sp-consensus-pow = { git = "https://github.com/paritytech/substrate", tag = 'v2.0.0-alpha.3' }
+sc-client-api = { git = "https://github.com/paritytech/substrate", tag = 'v2.0.0-alpha.3' }
+sp-blockchain = { git = "https://github.com/paritytech/substrate", tag = 'v2.0.0-alpha.3' }
+
+runtime = { package = "pow-runtime", path = "../../runtimes/pow-runtime" }
+runtime-genesis = { package = "pow-genesis", path = "../../runtimes/pow-genesis" }
+
+# Attempt to use the babe inherent that the super-runtime expects. This hack didn't work.
+# [dependencies.sp-consensus-babe]
+# git = "https://github.com/paritytech/substrate"
+# tag = 'v2.0.0-alpha.3'
```

Comment:
I think our style is not to merge commented out code. Unless you think this is useful.